src/com/amazon/corretto/crypto/provider/EcUtils.java

```diff
@@ -220,6 +220,18 @@ static String getNameByEncoded(final byte[] encoded) {
     return name;
   }
 
+  static boolean ecParameterSpecsAreEqual(final ECParameterSpec a, final ECParameterSpec b) {
+    if (a == b) {
+      return true;
+    } else if (a == null || b == null) {
```

Comment:
nit: i doubt this will come up in practice, but should this function return `true` if _both_`a` and `b` are null?

Reply:
Yes it should. This is both standard in Java and also matches the general assumption that the equality is symmetric and so having `ecParameterSpecsAreEqual(param, param)` ever *not* equal `true` would be an unwelcome surprise.